realtime/server.js

```diff
@@ -6,8 +6,10 @@ var winston = require('winston');
 winston.add(winston.transports.File, { filename: 'server.log' });
 winston.level = 'debug';
 
+// We obtain from common.json information like 'hostname' and 'post'
```

Comment:
Rephrase imperatively:

Obtain information like 'hostname' and 'port' from common.json